pkg/output/cli.go

```diff
@@ -95,7 +106,7 @@ func (w *OutputWriter) printFailures(results []runtime.TestResult) {
 			w.fprintf(r.Error.Error())
 			continue
 		}
-		if !r.Success {
+		if !r.Success && !r.Skipped {
```

Comment:
I would move the skipped check at the beginning of the for-loop because than you see directly that skipped test cases are ignored.

```
	for _, tr := range results {
		r := convertTestResult(tr)
                if r.Skipped {
                     continue
                }

		if r.Error != nil {
			w.fprintf(w.au.Bold(w.au.Red(w.template.errors(r))))
			w.fprintf(r.Error.Error())
			continue
		}
		if !r.Success {
			w.fprintf(w.au.Bold(w.au.Red(w.template.failures(r))))
			w.fprintf(r.Diff)
		}
	}
```